.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,56 @@
+name: Prerelease 🚀
+
+on:
+  push:
+    branches:
+      # Target main branch
+      - "main"
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  prerelease:
+    name: Changests Prerelease
+    if: github.repository == 'latticexyz/mud'
```

Comment:
@roninjin10 there's still a few of these here. Lmk if you want me to remove or

Reply:
```suggestion
```
I likely accidentally changed it on github and then force pushed it away